### ISSUE TITLE
AT support

### ISSUE THREAD

**Issue by Comandeer** [mm-jsr/jsr#2]
Slider should correctly expose its role for AT usage. It should also expose minimal and maximal value via appropriate ARIA attributes.

**Comment by soanvig** [comment]
ARIA attributes are going to be added along with screen readers support. After that code should be self-explaining.

Support for AT is planned before final out-of-beta release.
